### ISSUE TITLE
Limiting scrollback

### ISSUE THREAD

**Issue by tewe** [MacIrssi/MacIrssi#51]
Looks like irssi's scrollback setting are removed; it's always infinite. I idle in some channels where this is a serious waste of memory.

**Comment by ghost** [comment]
Indeed, there are irssi options for this that MacIrssi doesn't obey. I'll look into it.

**Comment by Dakta** [comment]
This is the same reason MacIrssi doesn't support irssi's `/lastlog` command: it's part of irssi's UI package for the command-line interface.

I'd like to see configurable scrollback with configurable logging for infinite scrollback, and support for `/lastlog`-style searches that start at the current line then go deeper into the scrollback buffer instead of the current search that starts at the top of the text field that MacIrssi uses for a scrollback buffer.
